component/src/test/java/org/wso2/extension/siddhi/execution/env/GetEnvironmentPropertyFunctionExtensionTestCase.java

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
Remove this leading line.

Reply:
fixed with 481b71d3a6bcc2879f0352e76915f3eedaaf3bc8